src/header/header.scss

```diff
@@ -429,3 +394,28 @@
     }
```

Comment:
these 2 classes are really confusing... `iui-button-icon` and `iui-header-button-icon`.
Seems disabled still has cursor pointer (in simple buttons, not split, which we dont have example for)

Reply:
Disabled cursor has been fixed.